fix typo (#4294)
docker-mailserver/docker-mailserver: docs/content/usage.md

- The **MX record** tells everyone which (DNS) name is responsible for e-mails on your domain.
    Because you want to keep the option of running another service on the domain name itself, you run your mail server on `mail.example.com`.
    This does not imply your e-mails will look like `test@mail.example.com`, the DNS name of your mail server is decoupled of the domain it serves e-mails for.

>     In theory, your mail server could even serve e-mails for `test@some-other-domain.com`, if the MX record for `some-other-domain.com` points to `mail.example.com`.

Reviewer: ```suggestion
    Your mail server could also handle emails for `test@some-other-domain.com`, if the MX record for `some-other-domain.com` points to `mail.example.com`.
```
